README.md

```diff
@@ -47,6 +47,7 @@ For details on multi-objective MDP's (MOMDP's) and other MORL definitions, we su
 |----------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|---------|------------------|--------------|-----------------------------------------------------------------------------------------------------------------------------|
 | [Envelope Q-Learning](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/envelope/envelope.py)                                      | Multi               | SER     | Continuous       | Discrete     | [Paper](https://arxiv.org/pdf/1908.08342.pdf)                                                                                        |
 | [PGMORL](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/pgmorl/pgmorl.py) *                                                     | Multi               | SER     | Continuous       | Continuous   | [Paper](https://people.csail.mit.edu/jiex/papers/PGMORL/paper.pdf) / [Supplementary materials](https://people.csail.mit.edu/jiex/papers/PGMORL/supp.pdf)        |
+| [Pareto Conditioned Networks](https://github.com/LucasAlegre/morl-baselines/blob/main/morl_baselines/multi_policy/pcn/pcn.py)                                        | Multi               | SER     | Continuous       | Discrete     | [Paper](https://www.ifaamas.org/Proceedings/aamas2022/pdfs/p1110.pdf)                                                          |
```

Comment:
Isn't it an ESR algorithm given that it learns deterministic policies to find the concave part of the front?

Reply:
It does not try to maximize any utility (scalarization) in specific, so I think it is useful for both cases actually?